Xamarin.Essentials/AppActions/AppActions.netstandard.tvos.watchos.macos.tizen.cs

```diff
@@ -6,8 +6,7 @@ namespace Xamarin.Essentials
 {
     public static partial class AppActions
     {
-        internal static bool PlatformIsSupported
-            => throw ExceptionUtils.NotSupportedOrImplementedException;
+        internal static bool PlatformIsSupported => false;
```

Comment:
This project uses NotSupportedOrImplementedException everywhere

Reply:
Yes, it does, and that's perfectly fine for methods like `AppActions.PlatformGetAsync` or `AppActions.PlatformSetAsync`: If I'm trying to get or set the AppActions on platforms that don't support it, I should clearly get an exception. But `AppActions.PlatformIsSupported` is different: It is asking whether the platform supports AppActions, and I want a yes-or-no answer here, not an exception.

An alternative to making `AppActions.IsSupported` public would be to wrap any usage of `AppActions` in try/catch blocks, but since the `IsSupported` method is already there, I find it easier to use it directly. It produces more readable code (and is probably more performant).

Reply:
Look at Vibration.IsSupported, for example. It's like that everywhere